client/web/src/repo/commits/GitCommitNodeTableRow.tsx

```diff
@@ -69,12 +69,14 @@ export const GitCommitNodeTableRow: React.FC<
 
     const commitMessageBody =
         expandCommitMessageBody || showCommitMessageBody ? (
-            <tr className={classNames(styles.tableRow, className)}>
-                <td colSpan={3}>
-                    <pre className={styles.messageBody}>
+            <tr className={classNames(styles.commitMessage, className)}>
+                <td className={classNames(styles.colByline)} />
+                <td>
+                    <div className={`${styles.messageBody} flex-1`}>
```

Comment:
nit: this looks like it's a little more idiomatic

```suggestion
                    <div className={classNames(styles.messageBody, 'flex-1')}>
```